lib/tasks/send_presentation_notifications.rake

```diff
@@ -0,0 +1,7 @@
+require 'rake'
+
+task send_presentation_notifications: :environment do
+  Event.requiring_notification.each do
+    EventNotificationJob.perform(event)
```

Comment:
Is that some Rails magic or something? I have a method defined as perform so I assumed that's what I had to call. Does it know about perform_later? 

Just looked up the docs. Yes it does? Weird.